core/system/src/context.c

```diff
@@ -87,7 +87,7 @@ static TContextPreviousState * context_state_pop(void)
 /* Return a pointer to the previous box context state. */
 TContextPreviousState * context_state_previous(void)
 {
-    if (!g_context_p) {
+    if (0 >= g_context_p) {
```

Comment:
please explain why the very same expression in lines 78 and 90 was replaced by different conditions?

Reply:
It was forgotten..